src/styles/blocks/ContainerWatch.scss

```
@@ -0,0 +1,72 @@
+
+.ContainerWatch {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  @include square (800px);
+  margin: auto;
+  background-color: cadetblue;
+  border-radius: 50%;
+}
+
+.ContainerWatch__minute {
```

Потрібно використовувати вкладеність, яку тобі дає SASS
```suggestion
&__minute {
```

---

done